src/wallet/mineTransaction.ts

```diff
@@ -81,6 +84,20 @@ class MineTransaction extends EventEmitter {
     });
   }
 
+  /**
+   * Used to handle errors in requests to the tx mining API 
+   * 
+   * @param error: The error that was received from the axiosInstance
+   */
+  handleRequestError(error: any) {
+    if (error.response && error.response.status === 429) {
+      this.emit('error', rateLimitExceededError);
+    } else {
+      console.error(error);
```

Comment:
Logging the error will give more info to anyone running a wallet about eventual problems. Especially for people running wallet-headless, because the error will be logged its stdout.